lib/compat/wordpress-6.1/class-wp-theme-json-6-1.php

```diff
@@ -801,6 +829,14 @@ protected static function compute_style_properties( $styles, $settings = array()
 			);
 		}
 
+		// If a variable value is added to the root, the corresponding property should be removed.
+		foreach ( $root_variable_duplicates as $duplicate ) {
+			$discard = array_search( $duplicate, array_column( $declarations, 'name' ), true );
+			if ( $discard ) {
```

Comment:
```suggestion
			if ( is_numeric( $discard ) ) {
```

Looks like it's letting `padding-top` through because `$discard === 0`

```json
body {
	padding-top: 12px;
	--wp--style--root--padding-top: 12px;
	--wp--style--root--padding-right: 45px;
	--wp--style--root--padding-bottom: 45px;
	--wp--style--root--padding-left: 25px;
}
```

Reply:
Oooh well spotted!